### PR TITLE
Sign online rpms before creating puddle

### PR DESCRIPTION
**Pull request by adammhaile**
@jupierce

### CODE REVIEW

File: build-scripts/sign_rpms.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -o xtrace
+
+TAG=${1}
+KEY=${2}
+BASE_TAG=`echo "${TAG}" | rev | cut -d '-' -f2- | rev`
+echo ${BASE_TAG}
+
+signdir=$(mktemp -d)
```
Comment (jupierce) [inline build-scripts/sign_rpms.sh]: Do we need to rm -rf after we run? You said sign_unsigned clears it out, but does it delete the directory entirely?

File: build-scripts/sign_rpms.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+pushd ${signdir}
```
Comment (jupierce) [inline build-scripts/sign_rpms.sh]: I didn't find it was necessary to be in the directory when I ran the command.

File: build-scripts/sign_rpms.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+builds=$(brew list-tagged --latest --quiet ${TAG} | awk '{print $1}')
+echo $builds
+
```
Comment (jupierce) [inline build-scripts/sign_rpms.sh]:
```
sign_script_path="/mnt/redhat/scripts/rel-eng/utility/sign_unsigned.py"
if [ ! -f "${sign_script_path}" ]; then
   echo "Unable to find rel-eng script: ${sign_script_path} . Make sure the tools drive is mounted."
   echo "In fstab: ntap-bos-c01-eng01-nfs01b.storage.bos.redhat.com:/devops_engineering_nfs/devarchive/redhat /mnt/redhat nfs tcp,ro,nfsvers=3 0 0"
   exit 1
fi
```

File: build-scripts/sign_rpms.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+/mnt/redhat/scripts/rel-eng/utility/sign_unsigned.py \
+	--workdir="${signdir}" \
+	--level="${KEY}" \
+	--exact --write-rpms \
+	--builds ${builds}
+
+popd
```
Comment (jupierce) [inline build-scripts/sign_rpms.sh]: Can be removed if pushd is removed.

File: build-scripts/sign_rpms.sh (hunk 1 above)
Comment (adammhaile) [inline build-scripts/sign_rpms.sh]: No... sign_unsigned.py calls `os.rmdir()` on the working directory we created after clearing it out.

File: build-scripts/sign_rpms.sh (hunk 2 above)
Comment (adammhaile) [inline build-scripts/sign_rpms.sh]: Fair enough.

File: build-scripts/sign_rpms.sh (hunk 3 above)
Comment (adammhaile) [inline build-scripts/sign_rpms.sh]: +1

File: build-scripts/sign_rpms.sh (hunk 4 above)
Comment (adammhaile) [inline build-scripts/sign_rpms.sh]: Done

**Comment by adammhaile** [comment]
@jupierce  pushed fixes for comments

**Pull request merged**